allennlp/modules/text_field_embedders/basic_text_field_embedder.py

```diff
@@ -0,0 +1,54 @@
+from typing import Dict
+
+from overrides import overrides
+import torch
+
+from allennlp.common import Params
+from allennlp.common.checks import ConfigurationError
+from allennlp.data import Vocabulary
+from allennlp.experiments import Registry
+from allennlp.modules import TextFieldEmbedder, TokenEmbedder
+
+
+@Registry.register_text_field_embedder("basic")
+class BasicTextFieldEmbedder(TextFieldEmbedder):
```

Comment:
In practice, what other ways apart from concatenation can you imagine connecting the outputs of `TokenEmbedders` together? I'm just wondering how useful this abstraction actually is.